public/scripts/controller.js

```diff
@@ -0,0 +1,104 @@
+var app = angular.module("app", []);
+app.controller("myController", function($scope) {
+
+        $scope.categories = ["Video", "BestPhoto", "Photo"];
+        $scope.addCategory = function () {
+            $scope.errortext = "";
+            if (!$scope.addMe) {return;}
+            if ($scope.categories.indexOf($scope.addMe) == -1) {
+                $scope.categories.push($scope.addMe);
+            } else {
+                $scope.errortext = "This category is already exists.";
```

Comment:
Error message is wrong.
It should be `This category already exists`, without verb.